std/datetime/date.d

```diff
@@ -2869,7 +2869,13 @@ public:
     {
         import std.format : format;
         try
-            return format("%sT%s", _date.toISOString(), _tod.toISOString());
+            return format(
+                "%sT%02d%02d%02d",
+                _date.toISOString(),
+                _tod._hour,
+                _tod._minute,
+                _tod._second
+            );
```

Comment:
I hate to get picky about formatting, but can you please either make these actually one line (which would definitely be preferable if it can be done within 120 characters), or add braces, since I really don't think that it's appropriate to skip braces when the body isn't actually a single line.